Puppetfile

```diff
@@ -6,10 +6,11 @@ mod 'benjamin/ramdisk', '0.0.1'
 mod 'camptocamp/augeas', '1.1.0'
 mod 'fsalum/redis'
 mod 'jamesnetherton/google_chrome', '0.4.0'
-mod 'puppetlabs/docker', '1.0.2'
 mod 'maestrodev/wget', '>= 1.1.0'
+mod 'nextrevision-automysqlbackup'
```

Comment:
Did you mean `/` instead of `-`?

Reply:
TIL that [it's the thing in the docs](https://forge.puppet.com/nextrevision/automysqlbackup). 😳